src/main/java/com/google/codeu/data/Message.java

```diff
@@ -44,13 +45,14 @@ public Message(UUID id, String user, String text, long timestamp, String recipie
     this.recipient = recipient;
   }
 
-  public Message(UUID id, String user, String text, long timestamp, String recipient, String imageUrl) {
+  public Message(UUID id, String user, String text, long timestamp, String recipient, String imageUrl, String imageLabels) {
```

Comment:
Can you figure out a way of modifying the constructor above so that you don't have to create a new constructor? Having both constructors( this one and the one above), is redundant.

Reply:
The two constructors are redundant, but this problem wasn't introduced by Elaine's commit - it is better if this is solved in another pull request, so I'm OK if Elaine doesn't do anything here. (But Elaine, if you want to fix this here, that's fine too, of course.)